### PR TITLE
display scoring for 'able to attend single ws' question

### PR DESCRIPTION
**Pull request by clareconstantine**
Able to attend single workshop:
screen shot 2018-01-15 at 2 10 57 pm

Able to attend multiple workshops:
screen shot 2018-01-16 at 1 53 43 pm

### CODE REVIEW

**Comment by aoby** [comment]
Looks like we still have some failing tests

File: dashboard/test/models/pd/application/teacher1819_application_test.rb [hunk 1]
```diff
@@ -187,6 +190,7 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
         {
           committed: Pd::Application::Teacher1819Application.options[:committed].last,
           able_to_attend_single: NO,
+          able_to_attend_multiple: Teacher1819ApplicationConstants::TEXT_FIELDS[:no_explain],
```
Comment (aoby) [inline dashboard/test/models/pd/application/teacher1819_application_test.rb]: note, you can `include Teacher1819ApplicationConstants` in the test and then reference `TEXT_FIELDS` directly. Either way is fine.

File: dashboard/test/models/pd/application/teacher1819_application_test.rb [hunk 2]
```diff
@@ -293,6 +300,49 @@ class Teacher1819ApplicationTest < ActiveSupport::TestCase
...
+          principal_approval: YES,
+          schedule_confirmed: NO,
+          diversity_recruitment: NO,
+          free_lunch_percent: '49.9%',
+          underrepresented_minority_percent: '49.9%',
+          wont_replace_existing_course: YES,
+          csd_which_grades: ['12'],
+          csd_course_hours_per_year: Pd::Application::ApplicationBase::COMMON_OPTIONS[:course_hours_per_year].last,
+          previous_yearlong_cdo_pd: ['Exploring Computer Science'],
+          taught_in_past: ['Exploring Computer Science']
```
Comment (aoby) [inline dashboard/test/models/pd/application/teacher1819_application_test.rb]: Are all these relevant to the test? Can we set it up more simply, just overriding `able_to_attend_multiple`?

**Comment by mehalshah** [comment]
The broken test failures are legit

**Pull request merged**